config/hsc/fgcmFitCycle.py

```diff
@@ -73,6 +73,17 @@
                             'N921': True,
                             'y': True,
                             'N1010': False}
+config.ccdGrayFocalPlaneDict = {'N387': True,
+                                'g': True,
+                                'r': True,
+                                'i': True,
+                                'N816': True,
+                                'z': True,
+                                'N921': True,
+                                'y': True,
+                                'N1010': True}
```

Comment:
At first, I was going to say that if all these filter Dicts are meant to stay in sync, it might be useful to define one and set each to that (will make updating easier/less error prone going forward).  However, I do see that `N387` and `N1010` are set to `True` here and `False` ...so I guess I'm now just making sure that was intentional?!

Reply:
These are intentional based on trial and error, that the shallow narrow-band filters just don't have enough data to fit reliably sub-ccd on a _per_ ccd level.  However, for the full focal plane fitting, these can be fit because the spatial scales are different. I'll add comments to the config.